stl/inc/utility

```diff
@@ -802,6 +818,27 @@ _NODISCARD constexpr underlying_type_t<_Ty> to_underlying(_Ty _Value) noexcept {
     _CSTD abort(); // likely to be called in debug mode, but can't be relied upon - already entered the UB territory
 #endif // _DEBUG
 }
+
+template <class _Ty, class _Uty>
+constexpr auto&& forward_like(_Uty&& _Ux) noexcept {

```

Comment:
The proposal says this should be `[[nodiscard]]`. Did you omit that for reason, or was it inadvertent?

Reply:
I'm sorry for just forgetting it. Perhaps because [`[[nodiscard]]` was absent in the first post on cppreference](https://en.cppreference.com/mwiki/index.php?title=std::forward_like&oldid=141395) of which I wasn't aware. I'm fixing this now.